client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -449,66 +461,70 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
             setAxiosInterceptorId(newInterceptor);
             axiosInterceptorId !== -1 && axios.interceptors.request.eject(axiosInterceptorId);
         }
-        setInvestigationStatus({
-            mainStatus: investigationRow.investigationStatus,
-            subStatus: investigationRow.investigationSubStatus
-        });
         const indexOfInvestigationObject = rows.findIndex(currInvestigationRow => currInvestigationRow.epidemiologyNumber === investigationRow.epidemiologyNumber);
         indexOfInvestigationObject !== -1 &&
             setCreator(rows[indexOfInvestigationObject].investigator.id);
-        if (investigationRow.investigationStatus === InvestigationMainStatus.NEW) {
+        if (investigationRow.investigationStatus === InvestigationMainStatus.NEW && shouldUpdateInvestigationStatus(investigationRow.investigatorId)) {
             logger.info({
-                service: Service.CLIENT,
+                ...logInfo,
                 severity: Severity.LOW,
-                workflow: 'Investigation click',
                 step: 'the user clicked a new investigation',
-                investigation: investigationRow.epidemiologyNumber,
-                user: user.id
             })
             axios.post('/investigationInfo/updateInvestigationStartTime', {
                 investigationStartTime: new Date(),
                 epidemiologyNumber: investigationRow.epidemiologyNumber
-            }).then(() => {
-                logger.info({
-                    service: Service.CLIENT,
-                    severity: Severity.LOW,
-                    workflow: 'Investigation click',
-                    step: 'updated investigation start time now sending request to update status',
-                    investigation: investigationRow.epidemiologyNumber,
-                    user: user.id
-                });
-                logger.info({
-                    service: Service.CLIENT,
-                    severity: Severity.LOW,
-                    workflow: 'Investigation click',
-                    step: `the investigator got into the investigation, investigated person: ${investigationRow.fullName}, investigator name: ${user.userName}, investigator phone number: ${user.phoneNumber}`,
-                    investigation: investigationRow.epidemiologyNumber,
-                    user: user.id
-                });
-                moveToTheInvestigationForm(investigationRow.epidemiologyNumber);
-            }).catch((error) => {
-                logger.error({
-                    service: Service.CLIENT,
-                    severity: Severity.LOW,
-                    workflow: 'GraphQL POST request to the DB',
-                    step: error
-                });
-                fireSwalError(OPEN_INVESTIGATION_ERROR_TITLE)
             })
+                .then(async () => {
```

Comment:
why having this func async?

Reply:
so i can await the promise returned from request to updateInvestigationStatus